### DISCUSSION TITLE
Troubleshooting OpenCV Version Conflict in Termux

### DISCUSSION THREAD

**Discussion by anishdevtech** [termux/termux-packages#17462] [Q&A] [answered]
# Problem
The current `opencv-python` version in the files is `4.8.0`, but when attempting to install other modules that require `opencv-python` using `pip` in Termux, the installation is denied. This is because on `pip`, version `4.8.0` does not exist, but `4.8.0.74` does.

# Solution
There are two possible solutions:

1. **Request Devs to Change the Version**: You can reach out to the developers of the module you want to install and request them to update their package requirements to allow for `opencv-python 4.8.0.74` instead of `4.8.0`.

2. **Manual Version Change**: If you are experienced and confident in handling system files, you can manually change the version of `opencv-python`. However, please note that modifying system files can be risky, and you should proceed with caution. Don't forget to create backups before making any changes.

# Question to Devs
To proceed with the manual version change, you'll need to know the location of the version file in your system. So, my question to the developers is: where does the version file of `opencv-python` exist in the system, so that I can manually make the necessary changes?

**Comment by sylirre** [comment, marked as answer]
Problem is not in versions. This is because module installed by `opencv-python` package is not visible by Pip.

**Reply by anishdevtech** [reply]
Tht maybe right  as pip can see numpy,matplotlib the issue is cv2 doesn't have a .dist-info file
